src/modules/message/utils/signMessageWithPrivateKey.js

```diff
@@ -1,7 +1,16 @@
 import { cryptography } from '@liskhq/lisk-client';
+import { sign } from 'tweetnacl';
 
 export const signMessageWithPrivateKey = ({ message, privateKey }) => {
   const result = cryptography.ed.signAndPrintMessage(message, Buffer.from(privateKey, 'hex'));
 
   return result;
 };
+
+export const signClaimMessageWithPrivateKey = ({ message, privateKey }) => {
+  const result = Buffer.from(
+    sign.detached(Buffer.from(message.substring(2), 'hex'), Buffer.from(privateKey, 'hex'))
```

Comment:
Wouldn't it be more reliable to perform a string replacement such as below instead of blindly using the substring?
```suggestion
    sign.detached(Buffer.from(message.replace(/^0x/, ''), 'hex'), Buffer.from(privateKey, 'hex'))
```

Reply:
The string is always started with `0x` based on the hashing from Portal.